library/aes.c

```diff
@@ -1075,7 +1077,9 @@ int mbedtls_aes_crypt_cbc( mbedtls_aes_context *ctx,
         while( length > 0 )
         {
             memcpy( temp, input, 16 );
-            mbedtls_aes_crypt_ecb( ctx, mode, input, output );
+            ret = mbedtls_aes_crypt_ecb( ctx, mode, input, output );
```

Comment:
I think it would be good to have some tests to inject errors into the function.

Reply:
I agree, but `mbedtls_aes_crypt_ecb()` always return 0

Reply:
I was actually imagining we replace `mbedtls_aes_crypt_ecb()` with a test equivalent to inject errors, but that's a bit ambitious and more of a test feature than a fix, and really outside the scope of this PR.

I can accept it as it is.